src/node_options.h

```diff
@@ -110,7 +110,7 @@ class EnvironmentOptions : public Options {
  public:
   bool abort_on_uncaught_exception = false;
   std::vector<std::string> conditions;
-  bool detect_module = false;
+  bool detect_module = true;
```

Comment:
I think we want to fix the format issue in one PR and then unflag the feature in the follow-up.

Reply:
I will split this into two PRs once the failing test with `__filename` usage is sorted out